src/main/java/org/killbill/billing/plugin/adyen/client/AdyenConfigProperties.java

```diff
@@ -317,7 +328,11 @@ public Boolean getAllowChunking() {
     }
 
     public String getPaymentUrl() {
-        return paymentUrl;
+        if (regionToPaymentUrlMap.isEmpty()) {
+            return paymentUrls;
+        } else {
+            return regionToPaymentUrlMap.get(currentRegion);
+        }
```

Comment:
In this logic, if we support multiple regions, but only have one in the map, we don't have any default. Instead:
```
String  regionPaymentUrl  = regionToPaymentUrlMap.get(currentRegion);
return regionPaymentUrl != null ? regionPaymentUrl : paymentUrls;
```